src/utils/mixins.scss

```
@@ -5,7 +5,7 @@
 }
 
 @mixin for-tablet() {
-  @media (min-width: 640px) {
+  @media (min-width: 640px) and (max-width: 1199px) {
```

```suggestion
  @media (min-width: 640px) {
```

Not neccessary for mobile first styling